samples/apps/bookinfo/src/productpage/Dockerfile.sidecar

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM python:2-onbuild
+
+RUN mkdir /opt/microservices
+RUN mkdir /opt/istio
+RUN mkdir /etc/istio
+RUN mkdir /etc/istio/proxy/
+RUN apt-get update && apt-get -y install iptables curl
+RUN adduser -disabled-password --gecos "" --uid 1337 istio
+RUN chown istio /etc/istio/proxy
+
+RUN set -ex \
+    && wget https://storage.googleapis.com/istio-build/proxy/envoy-debug-62755c63d9c80c043ece26504c4288845a929cc4.tar.gz \
```

Comment:
layer your docker so it's not copy pasted in multiple files?

Reply:
thanks, resolved.